sn_networking/src/record_store.rs

```diff
@@ -498,13 +498,8 @@ mod tests {
 
         assert!(store.put_verified(returned_record).is_ok());
 
-        // loop over store.get 10 times to make sure async disk write has had time to complete
```

Comment:
I don't think this is the right way to tackle this. 

We can compare the record returned with the original and only `break` then